static/js/login_or_signup.js

```diff
@@ -132,34 +132,34 @@ $(function() {
     };
 
     attachSubmitClickHandler(signupFormMetadata, (formDataAsJsonString) => {
-        $.post('/api/user', formDataAsJsonString, (responseBody, _, request) => {
-            if (request.status === 201) {
+        $.post('/api/user', formDataAsJsonString, (responseBody, _, jqXHR) => {
```

Comment:
Adrian and I have decided to use long winded variable names, so what's the long winded version of `jqXHR`. Reading online i suspect going by our conventions it should be `$XmlRequest` or `$XmlHttpRequest` The `$` signifies that its a jQuery object.

Reply:
I think `xhr` is one of those rare cases where the shorter name is actually more readable than the long name. Just because `xhr` is such a common thing in js and very quickly recognizable as "oh, it's one of _those_ objects". I expressed similar thoughts [here](https://github.com/atmiguel/cerealnotes/pull/16#discussion_r178424547). Basically, I prefer not to deviate from the commonly used names for things, even if that name isn't a name I would've chosen on my own.

Also fwiw, I got the name from the jquery docs. 
https://api.jquery.com/jquery.get/#jQuery-get-url-data-success-dataType

That being said, I like the idea of prepending jquery objects with `$`, so how do you feel about `$xhr`? If you feel strongly that it should be expanded then I prefer  `$XmlHttpRequest`, but it's a bit unfortunate because the whole reason I changed the name is because conceptually this isn't a request, it's a response. If I change it to `$XmlHttpRequest` then i'm just putting request back in the name. Not a huge deal because at least it's clear that it's an `xhr` object, it just bugs me.

Reply:
what about `$XmlHttpResponse`?

Reply:
Also i think Adrian's previous answer to your thoughts is still valid. For what its worth,  I had absolutely no idea what `jqxhr` was and had to look it up.